src/main/java/seedu/address/logic/commands/RemoveGradeCommand.java

```diff
@@ -0,0 +1,112 @@
+package seedu.address.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.commons.util.CollectionUtil.requireAllNonNull;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_ASSIGNMENT;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_NAME;
+
+import java.util.Map;
+import java.util.Objects;
+import java.util.Set;
+
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.assignment.Assignment;
+import seedu.address.model.person.Email;
+import seedu.address.model.person.Github;
+import seedu.address.model.person.Name;
+import seedu.address.model.person.Person;
+import seedu.address.model.person.Phone;
+import seedu.address.model.person.Telegram;
+import seedu.address.model.tag.Tag;
+
+/**
+ * Remove assignment grades from an existing person in the address book.
+ */
+public class RemoveGradeCommand extends Command {
+    public static final String COMMAND_WORD = "removeGrade";
+    public static final String MESSAGE_USAGE =
+            COMMAND_WORD
+                    + ": Removes a grade of an assignment from the person.\n"
+                    + "Parameters: "
+                    + PREFIX_NAME
+                    + "NAME "
+                    + PREFIX_ASSIGNMENT
+                    + "ASSIGNMENT "
+                    + "Example: "
+                    + COMMAND_WORD
+                    + " "
+                    + PREFIX_NAME
+                    + "John Doe "
+                    + PREFIX_ASSIGNMENT
+                    + "Ex09 ";
```

Comment:
Could you remove the extra whitespace so that it displays properly for the help command.